.sandbox/manifest.yaml

```diff
@@ -0,0 +1,8 @@
+hooks:
+  build:
+    cmd: 'npm install @vue/cli'
```

Comment:
For this template, I'd recommend using `post-checkout` in addition to `build`:

```yaml
hooks:
  post-checkout:
    cmd: 'npm install'
  build:
    cmd: 'npm run build'
```

`post-checkout` is the hook runs everytime the code base has been updated automatically with remote repo. The command here is often used for one time setup with a fresh/refreshed codebase.
`build` hook does the work for building/generating after some source code is changed, so this doesn't tie to code checkout/sync, and will run more frequently.